components/kyma-metrics-collector/pkg/process/process_test.go

```diff
@@ -247,8 +247,8 @@ func TestPopulateCacheAndQueue(t *testing.T) {
 		g.Expect(err).Should(gomega.BeNil())
 
 		for _, failedID := range provisionedAndDeprovisionedSubAccIDs {
-			runtime := kmctesting.NewRuntimesDTO(failedID, fmt.Sprintf("shoot-%s", kmctesting.GenerateRandomAlphaString(5)), kmctesting.WithProvisionedAndDeprovisionedState)
-			runtimesPage.Data = append(runtimesPage.Data, runtime)
+			skrRuntime := kmctesting.NewRuntimesDTO(failedID, fmt.Sprintf("shoot-%s", kmctesting.GenerateRandomAlphaString(5)), kmctesting.WithProvisionedAndDeprovisionedState)
```

Comment:
that variable name will introduce head scratches after the next product renaming

Reply:
changed to `rntme`